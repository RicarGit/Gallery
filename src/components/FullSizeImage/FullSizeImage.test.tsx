import { render } from '@testing-library/react'
import { FullSizeImage } from '.'

describe('FullSizeImage', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <FullSizeImage url='http://www.google.com/google' />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})