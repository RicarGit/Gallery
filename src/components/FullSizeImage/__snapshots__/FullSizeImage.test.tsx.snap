// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullSizeImage should match snapshot 1`] = `
<div
  class="styles__Container-sc-13z6nlr-0 kEEBKa"
>
  <a
    class="maximize"
    href="http://www.google.com"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="maximize icon"
      src="maximize.svg"
    />
  </a>
</div>
`;
